src/sentry/utils/metrics.py

```diff
@@ -85,7 +85,8 @@ def incr(self, key, instance=None, tags=None, amount=1):
 
 def incr(key, amount=1, instance=None, tags=None, skip_internal=True):
     sample_rate = settings.SENTRY_METRICS_SAMPLE_RATE
-    if not skip_internal and _should_sample():
+    if (not skip_internal and _should_sample() and
```

Comment:
100% no reason to coerce to a tuple here. That needlessly does a copy of the list each time.